app/templates/create.hbs

```diff
@@ -1,4 +1,9 @@
 <div class="ui container">
+  <div class="ui stackable grid">
```

Comment:
This will unconditionally show to the user. Where's the logic here?

Reply:
@iamareebjamal It's in unverified-user-message file itself.

Reply:
@iamareebjamal It's in unverified-user-message file itself.